docs/getting-started/kubernetes-quickstart.md

```diff
@@ -0,0 +1,155 @@
+# Getting started with tracee in Kubernetes 
+
+This guide is focused on running tracee on Kubernetes. To demonstrate how to use Tracee we will use minikube, a distribution of kubernetes that can run on your local machine.
+
+## Prerequisites 
+ 
+minikube - see installation instructions and dependencies [here](https://minikube.sigs.k8s.io/docs/start/). 
+ 
+Start minikube and ensure that it is running correctly:
+``` 
+minikube start
+
+```
+
+```
+[*] kubectl get po -A
+
+NAMESPACE     NAME                               READY   STATUS    RESTARTS   AGE 
+kube-system   coredns-565d847f94-kd9xx           1/1     Running   0          15s 
+kube-system   etcd-minikube                      1/1     Running   0          26s 
+kube-system   kube-apiserver-minikube            1/1     Running   0          26s 
+kube-system   kube-controller-manager-minikube   1/1     Running   0          26s 
+kube-system   kube-proxy-cvqjm                   1/1     Running   0          15s 
+kube-system   kube-scheduler-minikube            1/1     Running   0          26s 
+kube-system   storage-provisioner                1/1     Running   0          15s 
+```
+
+## Running tracee as a daemon set
+
+Tracee uses eBPF for host-based detections, meaning it has visibility into all processes running on a given host, regardless of them being separated into different containers or pods. Therefore, we want to have tracee running on each node.
+
+The following kubernetes manifest can be used to run tracee as a daemon set. This tells Kubernetes to have a single tracee process on each node.
+
+```
+--- 
```

Comment:
Point them to, instead of having the content here -> https://github.com/aquasecurity/tracee/blob/main/deploy/kubernetes/tracee/tracee.yaml